react/features/base/premeeting/functions.ts

```diff
@@ -266,6 +267,7 @@ export function isPreCallTestEnabled(state: IReduxState): boolean {
  */
 export function getPreCallICEUrl(state: IReduxState): string | undefined {
     const { prejoinConfig } = state['features/base/config'];
+    const { preCallTest } = state['features/dynamic-branding'];
 
-    return prejoinConfig?.preCallTestICEUrl;
+    return preCallTest?.iceUrl || prejoinConfig?.preCallTestICEUrl;
```

Comment:
Let's use ?? everywhere. It makes sense because it won't fall trough if the chosen value is false for example.

Reply:
My worry is that  `??` won't catch the `''` value and we tend to return `''` in the backend service if the value is missing. Of course we can change the backend service and actually there is an effort in the direction to return only what the client have set but even then I don't think it is a bad thing to guarded for `''`.

Reply:
If that's the case it might be better to filter the empty strings when we parse the backend response so we don't need to do check everywhere no?